pkg/ocm/client.go

```diff
@@ -120,6 +121,9 @@ func (b *ClientBuilder) Build() (result *Client, err error) {
 	builder := sdk.NewConnectionBuilder()
 	builder.Logger(logger)
 
+	// Add deprecation transport wrapper to automatically handle deprecation headers
+	builder.TransportWrapper(deprecation.NewTransportWrapper())
```

Comment:
We need to include this file's changes in a separate MR to get the pipeline passing I believe 🙂 The testing environment does not know this exists yet

Reply:
Sorry for the late reply @tzvatot - I miss GitHub notifications easily because I'm involved in some open source projects outside of work